pkg/controller/controllercmd/builder.go

```
@@ -207,7 +207,7 @@ func (b *ControllerBuilder) Run(ctx context.Context, config *unstructured.Unstru
 		if err := server.PrepareRun().Run(ctx.Done()); err != nil {
 			klog.Error(err)
 		}
-		klog.Fatal("server exited")
+		klog.Error("server exited")
```

async, changes the flow to ignore the error and not to exit the program